app/javascript/controllers/checkbox_with_nested_element_controller.js

```diff
@@ -0,0 +1,23 @@
+import { Controller } from "@hotwired/stimulus"
+
+/*
+ * CheckboxWithNestedElementController is a Stimulus controller that
+ * shows/hides a nested element when a checkbox is checked/unchecked.
+ */
+export default class extends Controller {
+  static targets = [ "nestedElement", "checkbox" ]
+
+  connect() {
+    this.toggleNestedElementVisiblity()
+  }
+
+  toggleNestedElementVisiblity() {
+    if (this.checkboxTarget.checked) {
+      this.nestedElementTarget.classList.remove("hidden")
+    } else {
+      this.nestedElementTarget.classList.add("hidden")
+    }
```

Comment:
Wanna try using [`toggle`](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle) directly?

Reply:
Sounds good! I'll give that a shot.